src/cpp/session/resources/schema/user-prefs-schema.json

```diff
@@ -1535,18 +1535,18 @@
             "title": "Save Retry Timeout",
             "description": "The maximum amount of seconds of retry for save operations."
         },
-        "insert_native_pipe_operator": {
-            "description": "Whether the Insert Pipe Operator command should insert the native R pipe operator, |>",
+        "use_native_pipe_operator": {
```

Comment:
@kevinushey @jgutman Since this preference key changed, do we care about a migration if it has been set?

Reply:
I didn't even think of that -- this name didn't really need to be changed, but since we wanted to use "UseNativePipeOperator" for the project setting I thought it would be easier to just be consistent across the code. I can change it back if it's problematic. I don't know how we do migrations though